### PR TITLE
home page reactjs initial commit

### LINKED ISSUE #2: Main page creation
create a react app inside frontend folder and organise things their properly

### PR DESCRIPTION
**Pull request by Aksh-Bansal-dev**
Screenshot (20)

### CODE REVIEW

**Review by MananJethwani — CHANGES_REQUESTED** [review]
aksh looks like this is not responsive can you please make it responsive using flexbox

**Comment by Aksh-Bansal-dev** [comment]
Screenshot (24)
Screenshot (25)

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+import HomePage from "./components/homePage";
+import Shortcuts from "./components/shortcuts";
+import Nav from "./components/Nav";
```
Comment (MananJethwani) [inline frontend/src/App.js]: @Aksh-Bansal-dev this should be "./components/nav" right?

File: frontend/src/components/homeNav/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import styles from './style.module.css';
+import NavLinks from "../navLinks";
+
+function HomeNav() {
+  return (
+    <div className={styles.container}>
+        <div className={styles.buttons}>
+        <span className={styles.navBtn}>About Us</span>
```
Comment (MananJethwani) [inline frontend/src/components/homeNav/index.jsx:8]: correct the indentation here

File: frontend/src/components/shortcuts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import styles from './style.module.css';
+
+
+function Shortcuts() {
+  
+  return (
+    <div className={styles.container}>
+        <div className={styles.logos}>
+        <a href="https://www.github.com"><img className={styles.logo}  src="./ComponentTMP_0-image2.png" alt="github"/></a>
```
Comment (MananJethwani) [inline frontend/src/components/shortcuts/index.jsx:9]: indentation here

File: frontend/src/components/svgbackground/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import styles from './style.module.css';
+
+function SvgBackground() {
+  return (
+    <>
+        <svg className={styles.waveSvg} xmlns="http://www.w3.org/2000/svg" width="833" height="1079" viewBox="0 0 833 1079">
+        <path id="Intersection_1" data-name="Intersection 1" d="M279.111,1207.79C322.137,1073.069,399.415,672.667,49.8,130.906V128.79H881.989l.811,24.761V1207.79Z" transform="translate(-49.8 -128.79)" fill="rgba(20,51,95,0.71)"/>
```
Comment (MananJethwani) [inline frontend/src/components/svgbackground/index.js:7]: indentation here

**Review by MananJethwani — APPROVED** [review]
correct the indentation and squash commits into one so that I can push it into main code

**Pull request merged**